Formula/m/minijinja.rb

```diff
@@ -0,0 +1,21 @@
+class Minijinja < Formula
+  desc "Render Jinja2 templates directly from the command-line to stdout"
+  homepage "https://github.com/mitsuhiko/minijinja"
+  url "https://github.com/mitsuhiko/minijinja/archive/refs/tags/2.3.1.tar.gz"
+  sha256 "c933a1d9f53de26bb776a61b9346e0b202c332b4ff1309db5d0178a9b603ca06"
+  license "Apache-2.0"
+  head "https://github.com/mitsuhiko/minijinja.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  def install
+    cd "minijinja-cli" do
+      system "cargo", "install", *std_cargo_args
+    end
+    bin.install "target/release/minijinja-cli" => "minijinja"
```

Comment:
If `cargo` installs it as `minijinja-cli`, why are we renaming it? It seems weird to get something different between `brew install` and `cargo install`.

Reply:
I understand this might have been brought up, I am fine with keeping it `minijinja-cli` but it seems neater naming it the library / repo name instead since it's a mono-repo for the library and cli tool.

Reply:
I have renamed this to minijinja-cli